tox.ini

```diff
@@ -14,3 +14,10 @@ commands =
 	{envpython} pycodestyle.py --statistics pycodestyle.py
 	{envpython} pycodestyle.py --doctest
 	{envpython} -m testsuite.test_all
+
+
+[testenv:flake8]
+deps =
+    flake8
+commands =
+    flake8 pycodestyle.py
```

Comment:
Flake8, as I'm sure you know, includes pyflakes. When I was writing some of the more recent pull requests, I found several pyflakes lint problems (which I also submitted separate pull requests for). If pycodestyle runs flake8 on all submitted changes, then those shouldn't pop up again.

It's something of an extra check to ensure the quality of pycodestyle doesn't dip.